core/src/components/input/input.ionic.scss

```diff
@@ -140,7 +140,7 @@
 // ----------------------------------------------------------------
 
 .input-bottom {
-  @include padding(7px, 0);
+  @include padding(globals.$ionic-space-100, 0);
```

Comment:
The Figma design looks like this should only apply padding to the top. Is this correct?

Reply:
Can confirm: https://github.com/ionic-team/ionic-framework/pull/29903/commits/4e3bd15d0bcfab65dcfc47be779b4e71e74cbc58